core-library/src/main/java/org/silverpeas/core/index/search/model/QueryDescription.java

```diff
@@ -277,7 +265,8 @@ public void setFieldQueries(List<FieldDescription> fieldQueries) {
 
   public boolean isEmpty() {
     return !StringUtil.isDefined(query) && getMultiFieldQuery() == null && getXmlQuery() == null &&
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of conditional operators (7) used in the expression (maximum allowed 3). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1067)